### DISCUSSION TITLE
[Model]Issue about printing a progress bar while training the model

### DISCUSSION THREAD

**Discussion by Joezxp** [ourownstory/neural_prophet#647] [Q&A - get help using NeuralProphet] [answered]
Hello, how should I set it so that the progress bar is not print when training the model? Thanks!
--------------------
OS:win10
Python Version 3.9.7

**Comment by ourownstory** [comment, marked as answer]
Might the `progress` or `minimal` args address your need?
https://neuralprophet.com/html/forecaster.html#neuralprophet.forecaster.NeuralProphet.fit

**Reply by Joezxp** [reply]
Thank you very much for your answer^_^.I set the "minimal" parameter before, but did not set the "progress" parameter, so the training process still has output. Now the problem is solved, thank you very much!
